mhcflurry/class1_affinity_prediction/train_allele_specific_models_command.py

```diff
@@ -82,32 +96,60 @@ def run(argv=sys.argv[1:]):
 
     predictor = Class1AffinityPredictor()
 
+    def train_fn(arguments):
```

Comment:
Style nit: we try to avoid abbreviations (the one exception is `num`). So this should be something like `train_function` (although function is kind of redundant here)